test/integration/operator_olm_test.go

```diff
@@ -44,19 +44,20 @@ var (
 
 func TestOLMIntegration(t *testing.T) {
 	if image, ok := os.LookupEnv(imageEnvVar); ok && image != "" {
-		defaultTestImageTag = image
+		testImageTag = image
 	}
-	t.Run("AnnotationsBasic", OperatorAnnotationsBasic)
-	t.Run("AnnotationsAllNamespaces", OperatorAnnotationsAllNamespaces)
-	t.Run("PackageManifestBasic", OperatorPackageManifestBasic)
+	t.Run("BundleBasic", OperatorBundleBasic)
+	t.Run("BundleAllNamespaces", OperatorBundleAllNamespaces)
+	t.Run("PackageManifestsBasic", OperatorPackageManifestsBasic)
+	t.Run("PackageManifestsMultiple", OperatorPackageManifestsMultiplePackages)
 }
 
```

Comment:
Shows that we have many tests to call the same functions, would be better to have one test with many scenarios. However, it is fine.

Reply:
Agreed, this should be refactored. I think we need to incorporate refactoring unit tests into our test planning.